docs/modules/configuring/con-configuring-network-addresses.adoc

```diff
@@ -155,12 +155,27 @@ clusterNetworkAddressConfigProvider:
   type: SniRoutingClusterNetworkAddressConfigProvider
   config:
     bootstrapAddress: mycluster.kafka.com:9192 # <1>                 
-    brokerAddressPattern: mybroker-$(nodeId).mycluster.kafka.com  
+    advertisedBrokerAddressPattern: mybroker-$(nodeId).mycluster.kafka.com  
```

Comment:
Do we document anything about deprecations in our `.adoc` files? I grepped and couldn't spot any mentions of deprecations.

Reply:
So far, no.  I think the CHANGELOG is good enough for down.  I think if we'd reach a 1.0, it would be a different story.

Reply:
its a good point to think about, but as @k-wall says we haven't done anything so far so lets stick to the CHANGELOG approach for now.